Auth0/Authentication.swift

```diff
@@ -358,6 +362,7 @@ public protocol Authentication: Trackable, Loggable {
      - parameter parameters: additional parameters sent during authentication
 
      - returns: a request that will yield Auth0 user's credentials
+     - warning: disabled for OIDC-conformant clients, an alternative will be added in a future release
```

Comment:
Add info of what grant type needs to be enabled and a link to the docs about grant type